### PR TITLE
DTFS2-5467 - Portal Bug - all facilities not showing as admin

### PR DESCRIPTION
**Pull request by Zainzzkk**
# Issue: 

* dashboard facilities not showing if logged in as admin as deal.bank.id was unknown

# Changes made:

* checked if superUser for setting bank.id
* updated tests

### CODE REVIEW

File: e2e-tests/portal/cypress/integration/journeys/dashboard/dashboard-facilities/dashboard-facilities-filters.spec.js [hunk 1]
```diff
@@ -56,6 +56,15 @@ context('Dashboard Deals filters', () => {
...
 
+    it('renders all facilities (Admin)', () => {
+      cy.login(ADMIN);
+      dashboardFacilities.visit();
+      dashboardFacilities.rows().should('be.visible');
+      dashboardFacilities.row.nameLink(ALL_FACILITIES[0]._id).should('exist');
+      dashboardFacilities.row.nameLink(ALL_FACILITIES[1]._id).should('exist');
+      dashboardFacilities.rows().should('have.length', ALL_FACILITIES.length);
+    });
+
```
Comment (ttbarnes) [inline e2e-tests/portal/cypress/integration/journeys/dashboard/dashboard-facilities/dashboard-facilities-filters.spec.js:67]: Can we move this to `journeys/admin/dashboard`?

**Review by ttbarnes — COMMENTED** [review]
Great stuff, if we could just move that e2e test spec please

File: e2e-tests/portal/cypress/integration/journeys/admin/fixtures/index.js [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+  bankInternalRefName: 'AIN BSS',
+};
+
+const BSS_DEAL_MIA = {
+  ...BSS_DEAL_BASE,
+  submissionType: CONSTANTS.DEALS.SUBMISSION_TYPE.MIA,
+  bankInternalRefName: 'MIA BSS',
+};
+
+const GEF_DEAL_DRAFT = {
```
Comment (IgnatG) [inline e2e-tests/portal/cypress/integration/journeys/admin/fixtures/index.js:49]: Are we using all these variables? If not, can we add only the things that we need?

Comment (Zainzzkk) [reply e2e-tests/portal/cypress/integration/journeys/admin/fixtures/index.js:19]: done

**Review by IgnatG — APPROVED** [review]

**Pull request merged**